### ISSUE TITLE
Allow programmatic use

### ISSUE THREAD

**Issue by HazCod** [psampaz/go-mod-outdated#10]
Hi,

First of all, thank you for your tool!
I have following two issues when using `go-mod-outdated -update -direct`.

1. Exit code is always zero, regardless of updates were found. There is no way to distinguish between updates found or not.

2. Empty table is shown when no updates were found.
```
+--------+---------+-------------+--------+------------------+
| MODULE | VERSION | NEW VERSION | DIRECT | VALID TIMESTAMPS |
+--------+---------+-------------+--------+------------------+
+--------+---------+-------------+--------+------------------+
```

Is it possible to e.g. return `-1` if nothing was found?

**Comment by psampaz** [comment]
Hi @HazCod thanks for pointing out these two issues. Please wait for #9

**Comment by psampaz** [comment]
#9 is now available on v0.3.0
